### PR TITLE
Add help text for explaining time of day that job listing will expire

### PR DESCRIPTION
**Pull request by jom**
### Changes proposed in this Pull Request

* Minor change to add help text for the time of day that a job listing will expire.

Screen Shot 2021-02-08 at 6 57 01 pm

### CODE REVIEW

File: includes/class-wp-job-manager-post-types.php [hunk 1]
```diff
@@ -1473,6 +1473,11 @@ public static function get_job_listing_fields() {
 			$application_method_placeholder = __( 'https://', 'wp-job-manager' );
 		}
 
+		$job_expires_description = __( 'Job listing expires at the end of the selected day.', 'wp-job-manager' );
```
Comment (alexsanford) [inline includes/class-wp-job-manager-post-types.php:1476]: I wonder if we could just say "... at the end of the day"?

File: includes/class-wp-job-manager-post-types.php [hunk 2]
```diff
@@ -1473,6 +1473,11 @@ public static function get_job_listing_fields() {
...
+		if ( ! self::instance()->jobs_expires_end_of_day() ) {
+			$job_expires_description = __( 'Job listing expires at the start of the selected day.', 'wp-job-manager' );
```
Comment (alexsanford) [inline includes/class-wp-job-manager-post-types.php:1478]: Same as above, could we just say "... at the start of the day"?

**Review by alexsanford — APPROVED** [review]
A small comment on wording, but otherwise LGTM

File: includes/class-wp-job-manager-post-types.php (hunk 1 above)
Comment (jom) [reply includes/class-wp-job-manager-post-types.php]: Yup, had that originally then I think I overthought it when in the interface. Changed.

**Pull request merged**